docs/dmsgpty.md

```diff
@@ -107,4 +109,74 @@ To start the `dmsgpty-ui` simply run
 $ ./bin/dmsgpty-ui
 ```
 
-And open the browser at http://127.0.0.1:8080/
\ No newline at end of file
+And open the browser at http://127.0.0.1:8080/
+
+### Connect Two Local DMSG Hosts with DMSGPTY
```

Comment:
We need a documentation for connecting two remote machines with each other. This is a rare case so it should not be presented as the main case. It is sufficient to explain for this case how to modify the standard setup.

Reply:
Okay will change accordingly